### PR TITLE
Create PDF from application data and Upload

### PR DESCRIPTION
**Pull request by JoeChapman**
Add the pdf behaviour, model, and upload model to create and upload a pdf to S3 that can be retrieved from a link emailed to the caseworker.
Note: This PR does not send the email

Run this with `docker-compose up`.

### CODE REVIEW

**Comment by sulthan-ahmed** [comment]
Looks like it's failing on the drone build `WARNING: unable to verify the Yaml signature.`

**Comment by JoeChapman** [comment]
@sulthan-ahmed should pass now.

File: test/unit/models/pdf.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+const config = require('../../../config');
+const proxyquire = require('proxyquire');
+const isPdf = sinon.stub();
+const Model = proxyquire('../../../apps/right-to-rent-check/models/pdf', {
+  'is-pdf': isPdf
+});
+
+describe.only('PDF Model', () => {
```
Comment (sulthan-ahmed) [inline test/unit/models/pdf.js]: do you want to remove the `.only`?

**Comment by sulthan-ahmed** [comment]
How do you run the app in dev mode? I get a `connect ECONNREFUSED 127.0.0.1:8001`

File: apps/right-to-rent-check/acceptance/features/declaration/submission.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const add = (n) => {
```
[the remark this answers is not in the document]
Comment (JoeChapman) [reply apps/right-to-rent-check/acceptance/features/declaration/submission.js]: nah it's fine

File: test/unit/models/pdf.js (hunk 1 above)
Comment (JoeChapman) [reply test/unit/models/pdf.js]: yes I need to remove that

**Comment by JoeChapman** [comment]
@sulthan-ahmed please use `docker-compose up` to run the app.

**Comment by JoeChapman** [comment]
Or run with `npm run dev:ci` so the mock apis are available.

**Comment by sulthan-ahmed** [comment]
thanks, I'll test it now

**Pull request merged**